lib/key-generators/version-commit.js

```diff
@@ -0,0 +1,37 @@
+var CoreObject  = require('core-object');
+var gitRepoInfo = require('git-repo-info');
+var fs          = require('fs');
+var Promise     = require('ember-cli/lib/ext/promise');
+
+var denodeify   = require('rsvp').denodeify;
+var readFile    = denodeify(fs.readFile);
+
+module.exports = CoreObject.extend({
+  init: function(options) {
+    this._plugin = options.plugin;
+  },
+
+  generate: function() {
+    var versionFile = this._plugin.readConfig('versionFile') || 'package.json';
```

Comment:
This default value should be configured as part of the plugin's `defaultConfig` property (https://github.com/zapnito/ember-cli-deploy-revision-key/blob/master/index.js#L14)